### ISSUE TITLE
Mac Template Pack snippets

### ISSUE THREAD

**Issue by MartinOpde** [PrismLibrary/Prism-Extensibility#32]
Did you remove the prism snippets?

After I upgraded to template pack version 2.0 I do no longer have snippets.

**Comment by brianlagunas** [comment]
Whoops... good catch.  Seems they got lost when we converted over to the new templates for VS for Mac.  We'll get those back in.

**Comment by MartinOpde** [comment]
Hi @brianlagunas, did you plan to get snippets back in ?

Thx.

**Comment by brianlagunas** [comment]
Yes, as my previous comment stated: 

> We'll get those back in.

**Comment by MartinOpde** [comment]
Yes I understood that but I meant "Did you plan it" as "will it be done soon?" because it is very helpful

**Comment by brianlagunas** [comment]
We do not provide time estimates. It will be done when we have time to get to it.  If you need them sooner, you may submit a pull request.

**Comment by MartinOpde** [comment]
Ok thx, by the way you are doing a great job !

**Comment by dansiegel** [comment]
or use the QuickStart
